pycbc/filter/qtransform.py

```diff
@@ -128,7 +209,7 @@ def qplane(qplane_tile_dict, fseries, frange, normalized=True, tres=0.001, fres=
     """
     # store q-transforms of each tile in a dict
     qplane_qtrans_dict = {}
-    dur = 1.0 / fseries.delta_f
+    dur = fseries.to_timeseries().duration
```

Comment:
frequency series have a duration property now, please use that instead of converting to the time domain.